torch/_inductor/codegen/common.py

```diff
@@ -320,6 +320,9 @@ def _print_GreaterThan(self, expr):
         # Go figure...
         return " >= ".join(map(self.paren, map(self._print, expr.args)))
 
+    def _print_Piecewise(self, expr):
+        # This is a hack, but we only ever emit `Piecewise` when converting bool to int
+        return f"bool({expr.args[0]}[0])"
```

Comment:
Hacky bit for emitting cleaner code. Alternative (non-specialized) is `expr.args[0][0] if expr.args[0][1] else expr.args[1][0]`